WalletWasabi/WabiSabi/Backend/WabiSabiConfig.cs

```diff
@@ -89,7 +89,7 @@ public WabiSabiConfig(string filePath) : base(filePath)
 	public CoordinationFeeRate CoordinationFeeRate { get; set; } = new CoordinationFeeRate(0.003m, Money.Coins(0.01m));
 
 	[JsonProperty(PropertyName = "CoordinatorExtPubKey")]
-	public ExtPubKey CoordinatorExtPubKey { get; } = Constants.WabiSabiFallBackCoordinatorExtPubKey;
+	public ExtPubKey CoordinatorExtPubKey { get; private set; } = Constants.WabiSabiFallBackCoordinatorExtPubKey;
```

Comment:
Re you sure a `private` setter can be deserialized?

Reply:
I have tested it. The current master always overwrites the config right after the start. With this PR it is not doing that.

Reply:
I considered this a success in fixing the issue.